server/metadata/tenant.go

```diff
@@ -601,7 +610,7 @@ func (tenant *Tenant) GetNamespace() Namespace {
 // metadata version once the commit is successful so reloading happens at the next call when a transaction sees a stale
 // tenant version. This applies to the reloading mechanism on all the servers. It returns "true" If the database already
 // exists, else "false" and the error.
-func (tenant *Tenant) CreateDatabase(ctx context.Context, tx transaction.Tx, dbName string) (bool, error) {
+func (tenant *Tenant) CreateDatabase(ctx context.Context, tx transaction.Tx, dbName string, accessToken *types.AccessToken) (bool, error) {
```

Comment:
What about passing DatabaseMetadata as a param and then we can have a helper method to build DatabaseMetadata which is called by the runner? This way if we add anything to the database metadata then we don't need to change the function signature.

Reply:
sounds good. made relevant updates.